SA3/qa_web/models.py

```diff
@@ -3,8 +3,35 @@
 from django.contrib.auth.models import AbstractUser
 
 class User(AbstractUser):
-    pass
-    
+    """
+       User will be able to insert his or her personal information,
+       this will then be uploaded and saver the appropriate database.
+       Furthermore, these fields are added to the pre-existing fields provided
+       by Django such as first name, last name, email.
+
+     """
+
+    age = models.IntegerField(null =True)
+    birthday = models.DateField(null = True)
+    motherland=models.TextField(max_length=100, null =True)
+    school = models.TextField(max_length=100, null =True)
+    major = models.CharField(max_length=50, null =True)
+    city = models.TextField(max_length=100, null=True)
+
+    """
+      The below code has a purpose that a user profile will be created when
+      a user is created
+
+     """
+
+    def create_profile(self):
+        pass
```

Comment:
Is there even use to declaring this method?

Reply:
nope, it slid through my reviewing, I removed it now.